administrator/language/en-GB/en-GB.com_joomlaupdate.ini

```
@@ -24,6 +24,8 @@ COM_JOOMLAUPDATE_MINIMUM_STABILITY_DEV="Development"
 COM_JOOMLAUPDATE_MINIMUM_STABILITY_LABEL="Minimum Stability"
 COM_JOOMLAUPDATE_MINIMUM_STABILITY_RC="Release Candidate"
 COM_JOOMLAUPDATE_MINIMUM_STABILITY_STABLE="Stable"
+COM_JOOMLAUPDATE_MSG_WARNINGS_UPLOADNOTENABLED="Joomla Update can't continue until file uploads are enabled for the server."
+COM_JOOMLAUPDATE_MSG_WARNINGS_ZLIBMISSING="Joomla Update can't continue until Zlib is installed."
```

This needs to be clear that you are referring to something on the server and not an extension from JED ;)

---

It means "Zlib php extension is missing on the server"

---

The strings are exactly the same which are here since ages for the installer, only that "Installer" has been replaced by "Joomla Update". So if it was wrong, it was wrong all the time, and @brianteeman could have made a PR long time ago.

---

If I had seen it I would have changed it ;)
there are a lot of language strings I can't be expected to have read them all.

```suggestion
COM_JOOMLAUPDATE_MSG_WARNINGS_ZLIBMISSING="Joomla Update can't continue until Zlib is enabled for the server."
```

---

Shall I change `COM_INSTALLER_MSG_INSTALL_WARNINSTALLZLIB` for the installer in the same way so it's consistent? The string is not touched by this PR, but that's the string where the above one was derived from.

---

please

---

Ok, will do so. Thanks, marking as resolved as soon as commited.

---

Done.